test/e2e/cases/05-ansiblejob/00-secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: toweraccess
+  namespace: default
+type: Opaque
+stringData:
+  token: _token_here
+  host: _awx_ip_here_
```

Comment:
@mikeshng  I am wondering where the two macro variables are replaced. Also the ansible job operator is not installed.  Is there another script for setting up the ansible tower secret and installing the ansible job operator?

Reply:
@xiangjingli I have a comment in the PR and the commit message that this is just a test that `Its only responsible for verifying that ansiblejob prehook CR gets created and the CR name is in the subscription status.`

Reply:
The GitHub action KinD is probably not going to be powerful enough to host an actually running instance of AnsibleTower or AWX. In my opinion, checking the AnsibleJob CR gets created is a good start.

Reply:
Gotcha. yeah it is a good start